arduino/resources/helpers_test.go

```diff
@@ -16,7 +16,7 @@
 package resources
 
 import (
-	"io/ioutil"
+	"os"
 	"net/http"
 	"net/http/httptest"
```

Comment:
```suggestion
	"net/http"
	"net/http/httptest"
	"os"
```

Resolve the `go fmt` violation that causes the "Check Go" CI workflow run failure:

https://github.com/arduino/arduino-cli/actions/runs/4039005652/jobs/6943394036